app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1030,6 +1030,20 @@ def validate(self, data):
         return data
 
     def update(self, experiment, validated_data):
+        if (
+            experiment.is_rollout
+            and not experiment.is_paused
+            and experiment.status == NimbusExperiment.Status.LIVE
+            and experiment.status_next is None
+            and experiment.publish_status == NimbusExperiment.PublishStatus.IDLE
+            and validated_data.get("publish_status") == NimbusConstants.PublishStatus.IDLE
```

Comment:
It's probably worth adding a parameterized test to test a bunch of these cases?  No need to be exhaustive but like, a case where it matches everything and then mutate the match everything case for each of those fields and assert false is probably sufficient?

Reply:
Great idea!! 💯